Packs/AWS-Enrichment-Remediation/Scripts/AWSAccountHierarchy/AWSAccountHierarchy.py

```diff
@@ -0,0 +1,151 @@
+                return "NONE", temp
+            root = root_info.get("AWS.Organizations.Root(val.Id && val.Id == obj.Id)")[
+                0
+            ]
+            temp["level"] = str(level)
+            temp["id"] = root.get("Id", "")
+            temp["name"] = root.get("Name", "")
+            temp["arn"] = root.get("Arn", "")
+        else:
+            raise ValueError("unexpected object type")
+    except TypeError:
+        return "NONE", temp
+    else:
+        return next_one, temp
+
+
+""" COMMAND FUNCTION """
+
+
+def aws_account_heirarchy(args: Dict[str, Any]) -> CommandResults:
+    """
+    Determine AWS account hierarchy by looking up parent objects until the root level is reached.
+    Args:
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        list[CommandResults]: outputs, readable outputs and raw response for XSOAR.
+
+    """
+
+    account_id = args.get("account_id")
+
+    if not account_id:
+        raise ValueError("account_id not specified")
+    # Using `demisto.executeCommand` instead of `execute_command` because for
+    # multiple integration instances we can expect one too error out.
+    account_info = demisto.executeCommand(
+        "aws-org-account-list", {"account_id": account_id}
+    )
+    account_returned = [
+        account
+        for account in account_info
+        if (not isError(account) and account.get("Contents").get("Arn"))
+    ]
+    if not account_returned:
+        return CommandResults(readable_output="could not find specified account info")
+    else:
+        match_account = account_returned[0].get("Contents")
+        instance_to_use = account_returned[0]["Metadata"]["instance"]
+    level = 1
+    hierarchy = [
+        {
+            "level": "account",
+            "id": match_account.get("Id", ""),
+            "name": match_account.get("Name", ""),
+            "arn": match_account.get("Arn", ""),
+        }
+    ]
+    account_parent = demisto.executeCommand(
+        "aws-org-parent-list", {"child_id": account_id, "using": instance_to_use}
+    )
+    if isError(account_parent):
+        return CommandResults(readable_output="could not find specified account parent")
+    next_one, to_append = lookup(
+        account_parent[0].get("Contents", {})[0].get("Id", ""), level, instance_to_use
+    )
+    if next_one == "NONE":
+        return CommandResults(readable_output="could not find specified ou/root info")
+    hierarchy.append(to_append)
+    try:
+        while "stop" not in next_one:
+            level += 1
+            next_one, to_append = lookup(next_one, level, instance_to_use)
+            if next_one == "NONE" or next_one is None:
+                return CommandResults(
+                    readable_output="could not find specified ou/root info"
+                )
+            hierarchy.append(to_append)
+    except TypeError:
+        return CommandResults(readable_output="could not find specified ou/root info")
+
```

Comment:
Let's export this part to a separated function

Reply:
this automation is modeled off this one: https://github.com/demisto/content/tree/master/Packs/GCP-Enrichment-Remediation/Scripts/GCPProjectHierarchy

strongly prefer to keep it the same to keep the automations the same.